### PR TITLE
Add benchmark test

### PR DESCRIPTION
**Pull request by choo-stripe**

### CODE REVIEW

File: pkg/v1/remote/write_test.go [hunk 1]
```diff
@@ -1269,3 +1269,30 @@ func TestTag(t *testing.T) {
...
+			b.Fatalf("random.Image(...): %v", err)
+		}
+
+		tagStr := strings.TrimPrefix(s.URL+"/test/image:tag", "http://")
+		tag, err := name.NewTag(tagStr)
+		if err != nil {
+			b.Fatalf("parsing tag (%s): %v", tagStr, err)
+		}
+
+		err = Write(tag, img)
```
Comment (jonjohnsonjr) [inline pkg/v1/remote/write_test.go]: We probably want to `b.ResetTimer()` right before this, if that's what we're measuring.

**Comment by choo-stripe** [comment]
> Are you able to sign the CLA?

@jonjohnsonjr I'm pretty sure stripe has signed google's CLA. I'm not sure what to put for "Authorized Contributors Group". Is this something I need to reach out about internally or something on your end?

**Comment by jonjohnsonjr** [comment]
> I'm pretty sure stripe has signed google's CLA. I'm not sure what to put for "Authorized Contributors Group". Is this something I need to reach out about internally or something on your end?


I'm not sure. I've asked someone internally and will let you know when I hear back.

**Comment by jonjohnsonjr** [comment]
They said to try contacting `carl@<your domain>` (Eng) or `de@<your domain>` (IP Counsel).

File: pkg/v1/remote/write_test.go [hunk 2]
```diff
@@ -1269,3 +1269,32 @@ func TestTag(t *testing.T) {
 		t.Errorf("Validate() = %v", err)
 	}
 }
+
+func BenchmarkWrite(b *testing.B) {
+	// unfortunately the registry _and_ the img have caching behaviour, so we need a new registry
+	// and image every iteration of benchmarking.
+	for i := 0; i < b.N; i++ {
+		// set up the registry
+		s := httptest.NewServer(registry.New())
```
[the remark this answers is not in the document]
Comment (ImJasonH) [reply pkg/v1/remote/write_test.go]: Oh my mistake, `registry` only persists to memory, so it shouldn't be too slow. If we ever support other backing stores for `registry` we should consider having this benchmark use `ioutil.Discard`.

**Comment by choo-stripe** [comment]
@googlebot I signed it!

**Comment by jonjohnsonjr** [comment]
Thanks!

**Pull request merged**